tensorflow/core/grappler/optimizers/arithmetic_optimizer.cc

```diff
@@ -2322,7 +2322,7 @@ class FoldTransposeIntoMatMul : public ArithmeticOptimizerStage {
   }
 };
 
-// Fold Transpose into matrix multiplication.
+// Fold Conjugate into Transpose.
```

Comment:
Probably this comment should just be deleted?  It is only repeating the class name with spaces. :)